static/bidder-info/rise.yaml

```diff
@@ -8,6 +11,7 @@ capabilities:
     mediaTypes:
       - banner
       - video
+      - native
```

Comment:
Adding a new media type requires accompanying json tests with requests for that new media type.

Reply:
Added a native JSON test